### PR TITLE
PLAT-133 make new binary insolard_headless

### PR DESCRIPTION
**Pull request by aalekseev-ru**
make new binary insolard_headless that runs nodes without role logic. NB - you still need at least 1 heavy material role

I will add support for launchnet.sh in next task

### CODE REVIEW

File: ledger-core/v2/cmd/insolard_headless/main.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import (
+	"github.com/spf13/cobra"
+	jww "github.com/spf13/jwalterweatherman"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/log/global"
+	"github.com/insolar/assured-ledger/ledger-core/v2/server"
+	"github.com/insolar/assured-ledger/ledger-core/v2/version"
+)
+
+func main() {
```
Comment (AndreyBronin) [inline ledger-core/v2/cmd/insolard_headless/main.go]: Headless network is not a separate binary, it's a mode of insolard binary. See AppController doc

Comment (aalekseev-ru) [reply ledger-core/v2/cmd/insolard_headless/main.go]: my task is named as "headless network in SEPARATE binary", so I separated them. If it needed to be only a mode I can change this behavior

File: ledger-core/v2/network/gateway/base.go [hunk 2]
```diff
@@ -200,6 +198,8 @@ func (g *Base) createOriginCandidate() error {
 
 func (g *Base) StartConsensus(ctx context.Context) error {
 
+	// one of the nodes has to be in consensus.ReadyNetwork state,
+	// all other nodes has to be in consensus.Joiner
```
Comment (ruz) [inline ledger-core/v2/network/gateway/base.go]: this comment should be inside `if`, I've spend 5 minutes re-reading the comment to understand meaning until I realized that it describes code inside `if`

File: ledger-core/v2/server/internal/headless/components.go [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+		cryptographyService,
+		keyProcessor,
+	}...)
+
+	// TODO debug
+	// for _, componentMeta := range components {
+	// 	fmt.Print(componentMeta)
+	// 	reflect.ValueOf(componentMeta).Elem()
+	// }
+	cm.Inject(components...)
```
Comment (ruz) [inline ledger-core/v2/server/internal/headless/components.go]: delete this commented out code

**Pull request merged**